configdb/hardware/migrations/0033_alter_instrument_state.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.0.3 on 2023-01-09 23:18
+
```

Comment:
Migration part 1 just renames the integer state field to `state_int` and adds a new CharField `state`